docs/connecting-data/connecting-to-apis-and-external-data/affinity-api.md

```diff
@@ -4,4 +4,148 @@ description: How to Connect DataDistillr to the Affinity API
 
 # Affinity API
 
-Details coming soon!
\ No newline at end of file
+### Cost
+
+You can [contact](https://www.affinity.co/request-demo?utm_campaign=Capterra&utm_source=capterra&utm_medium=ppc){target=_blank} Affinity for a free demo and pricing.
+
+### Rate Limits
```

Comment:
Create a block with this like was done in ClickUp PR

Reply:
Same as last